File: 04_metodos_busca/01_busca_sequencial/busca_sequencial.go
package main

import "fmt"

func main() {
	list := []int{1, 2, 3, 4, 5, 6, 7, 8, 9, 10}

	fmt.Println(busca_sequencial(list, 7))
}

func busca_sequencial(list []int, search int) int {

	if len(list) < search {
		return -1
	} else {
		for i := 0; i <= len(list); i++ {
			if list[i] == search {
				return i
			}
		}
	}
	return -1
}

File: 04_metodos_busca/01_busca_sequencial/busca_sequencial.py
#!/usr/bin/python3
# -*- coding: utf-8 -*-

def busca_sequencial(list, search):
    i = 0

    while i < len(list):
        if list[i] == search:
            return i
        i += 1
    return False


list = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]

print(busca_sequencial(list, 10))